src/main/java/com/sprint/mission/discodeit/service/jcf/JCF_message.java

```diff
@@ -0,0 +1,66 @@
+package com.sprint.mission.discodeit.service.jcf;
+
+import com.sprint.mission.discodeit.entity.Channel;
+import com.sprint.mission.discodeit.entity.Message;
+import com.sprint.mission.discodeit.entity.User;
```

Comment:
unused import는 삭제하는게 좋을 것 같아요.  
```suggestion
import com.sprint.mission.discodeit.entity.Channel;
import com.sprint.mission.discodeit.entity.User;
```